playground/src/index.tsx

```diff
@@ -7,7 +7,11 @@ import './index.css';
 declare const COMMIT_ID: string;
 console.info(COMMIT_ID);
 
-ReactDOM.render(
-  <App />,
-  document.getElementById('root') as HTMLElement
-);
+const rootDiv: HTMLElement = document.getElementById('root') as HTMLElement;
+
+rootDiv.style.margin = '0';
+rootDiv.style.height = '100%';
+rootDiv.style.display = 'flex';
+rootDiv.style.flexDirection = 'column';
```

Comment:
I would recommend we set these styles in `index.css` since `<div id="root"></div>` is present on initial load of `index.html` and these styles are static (currently). This way the browser can use them during initial paint.

```css
#root {
  margin: 0;
  height: 100%;
  display: flex;
  flex-direction: column;
}
```

Reply:
I originally tried that, but it didn't work for some reason.  I couldn't figure out why.

---
In reply to: [223819332](https://github.com/Microsoft/tsdoc/pull/97#discussion_r223819332) [](ancestors = 223819332)